src/connectors/dashradio.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+Connector.playerSelector = '#music-dataview-container';
+
+Connector.getArtist = () => {
+	return Util.getTextFromSelectors('#artist-text').substring(3);
```

Comment:
If you want to remove "By" prefix, I suggest to use MetadataFilter.

See the `focusatwill` connector for example.

Reply:
Switched to that, thanks!